feature/mypage/src/main/java/com/mument_android/mypage/fragment/NoticeFragment.kt

```diff
@@ -34,18 +39,40 @@ class NoticeFragment : Fragment() {
         binding.myPageViewModel = myPageViewModel
 
         setNoticeRecyclerView()
+        itemClickEvent()
     }
 
 
     //공지사항 리싸이클러뷰
-    private fun setNoticeRecyclerView(){
+    private fun setNoticeRecyclerView() {
         noticeAdapter = NoticeAdapter()
         binding.rvNotice.adapter = noticeAdapter
 
-        myPageViewModel.noticeList.observe(viewLifecycleOwner){
-            noticeAdapter.submitList(it)
-        }
+        noticeList.add(
+            NoticeData(
+                0,
+                "공지사항1",
+                "공지사항 1 내용입니다.공지사항 1 내용입니다.공지사항 1 내용입니다.공지사항 1 내용입니다.공지사항 1 내용입니다.공지사항 1 내용입니다. \"https://www.naver.com/\"",
+                "2023-02-02"
+            )
+        )
+        noticeAdapter.submitList(noticeList)
     }
 
-
+    //아이템 클릭 리스너
+    private fun itemClickEvent() {
+        noticeAdapter.setItemClickListener(object : NoticeAdapter.OnItemClickListener {
+            override fun onClick(data: NoticeData) {
+                val title = data.title
+                val createdDate = data.created_at
+                val content = data.content
+                val intent = Intent(context, NoticeDetailActivity::class.java).apply {
```

Comment:
- Fragment 상에서 context는 Context? 타입이기 때문에 해당 방법을 사용해주세요!
```suggestion
                val intent = Intent(requireActivity(), NoticeDetailActivity::class.java).apply {
```

Reply:
ㅎㅋㅋㅎ제가 왜그랬을까요 넵넵!